tajo-catalog/tajo-catalog-client/src/main/java/org/apache/tajo/catalog/AbstractCatalogClient.java

```
@@ -140,13 +143,14 @@ public final Boolean existTablespace(final String tablespaceName) {
   }
 
   @Override
-  public TablespaceProto getTablespace(final String tablespaceName) {
+  public TablespaceProto getTablespace(final String tablespaceName) throws UndefinedTableException {
```

This method should throw UndefinedTablespaceException.